internal/rbd/rbd_util.go

```diff
@@ -230,6 +241,7 @@ func (rv *rbdVolume) openIoctx() error {
 		return err
 	}
 
+	ioctx.SetNamespace(pOpts.RadosNamespace)
```

Comment:
I think this needs to be 
```go
ioctx.SetNamespace(rv.RadosNamespace)
```

Currently the build fails because `pOpts` is undefined in this function.